lib/rubocop/cop/sorbet/block_method_definition.rb

```diff
@@ -61,6 +95,15 @@ def on_block(node)
 
         private
 
+        def in_activesupport_concern_class_methods_block?(node)
+          return false unless activesupport_concern_class_methods_block?(node)
+
+          immediate_module = node.each_ancestor(:module).first
+          return false unless immediate_module
```

Comment:
What does this test for given the 2 node matchers?

Reply:
hmmm, maybe I don't need it. Let me write a test and remove.